### DISCUSSION TITLE
react-native-svg not working with react native web

### DISCUSSION THREAD

**Discussion by Prashant-Mobile-Server** [software-mansion/react-native-svg#2204] [Q&A]
Getting error when trying to load svg image from a local directory. It is working fine on mobile but has below issue when I try to run for web.
Uncaught runtime errors:

×
**ERROR**
The string contains invalid characters.
createElement@[native code]
createElement@
createInstance@
completeWork@
completeUnitOfWork@
performUnitOfWork@
workLoopSync@
renderRootSync@
recoverFromConcurrentError@
performConcurrentWorkOnRoot@
workLoop@
flushWork@
performWorkUntilDeadline@

**### Made below changes to support on mobile:**

**custom.d.ts:**
`declare module '*.svg' {
  const content: React.FunctionComponent<React.SVGAttributes<SVGElement>>;
  export default content;
}
`
**tsconfig.json:**
{
  "compilerOptions": {
    "jsx": "react"
  },
  "extends": "@tsconfig/react-native/tsconfig.json",

  "include": ["**/*.ts", "**/*.tsx", "custom.d.ts"]
}

**Comment by EntitiCoder** [comment]
I have the same issues, what is your solution after, replace every direct import of svg ?
